scale/src/main/java/com/strategyobject/substrateclient/scale/ScaleUtils.java

```diff
@@ -30,6 +32,17 @@ public static <T> byte[] toBytes(T value, @NonNull ScaleWriter<T> writer) {
         return stream.toByteArray();
     }
 
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    public static <T> Injector<Class<?>, ScaleWriter, byte[]> toBytes(T value,
```

Comment:
I would suggest to return `byte[]` from the method.

Reply:
good idea